esp/logging/logginglib/logthread.cpp

```diff
@@ -195,7 +196,7 @@ bool CLogThread::enqueue(IEspUpdateLogRequestWrap* logRequest, const char* guid)
             GUID.set(guid);
         logRequest->setGUID(GUID.str());
         if (serializeLogRequestContent(logRequest, reqBuf))
-            logFailSafe->Add(GUID, reqBuf.str(), nullptr);
+            logFailSafe->Add(GUID, nullptr, reqBuf.str(), nullptr);
```

Comment:
If logRequest contains scriptValues, should they be propagated instead of passing nullptr?

Reply:
I think it is not needed. This part of code can only be reached for non decoupled logging. For non decoupled logging, all of log requests in tank files should be logged. If a log request should be skipped, the request will not be added into the tank file (already skipped in log manager).